css/style.css

```diff
@@ -629,7 +629,7 @@ div.tree ul {
 
     div.tree ul li.disabled {
       opacity: 0.65;
-      filter: url(effects.svg#fog);
+      /* filter: url(effects.svg#fog); */
```

Comment:
Remove these rather than comment them out; we have them in the git history should we ever need them :wink: